README.md

```diff
@@ -53,3 +54,4 @@ allowed_to_redefine_list = page,page2  # VDR311
 11. [VDR310. Some steps should not have an assertion](./flake8_vedro/rules/VDR310.md)
 12. [VDR311. Scope variables should not be redefined](./flake8_vedro/rules/VDR311.md)
 13. [VDR312. Scope variables should not be partially redefined](./flake8_vedro/rules/VDR312.md)
+14. [VDR313. Contexts should have specific assertions](./flake8_vedro/rules/VDR313.md)
```

Comment:
This rule is not about scenario -> should be in another block (like Contexts Rules) and have different code like VDR400